website/content/en/troubleshooting/_index.md

```diff
@@ -4,5 +4,68 @@ title: "Troubleshooting"
+
+### Isolate errors from logs
+
+You can use `grep ^E` and `--since` flag from `kubectl` to isolate any errors occurred after a given duration.
+
+**For Azure Key Vault provider versions less than 0.0.9**
+
+To troubleshoot issues with the csi driver and the provider, you can look at logs from the `secrets-store` container of the csi driver pod running on the same node as your application pod:
+
+```bash
+# find the secrets store csi driver pod running on the same node as your application pod
+kubectl get pods -l app=secrets-store-csi-driver -o wide
+kubectl logs <driver pod name> secrets-store --since=1h | grep ^E
+```
+
+**For Azure Key Vault provider versions equal to and greater than 0.0.9**
+
+For `0.0.9+` the provider logs are available in the provider pods. To troubleshoot issues with the provider, you can look at logs from the provider pod running on the same node as your application pod
+
+```bash
+# find the csi-secrets-store-provider-azure pod running on the same node as your application pod
+kubectl get pods -l app=csi-secrets-store-provider-azure -o wide
+kubectl logs <provider pod name> --since=1h | grep ^E
+```
+
+**For CSI driver logs**
+
+```bash
+# find the secrets store csi driver pod running on the same node as your application pod
+kubectl get pods -l app=secrets-store-csi-driver -o wide
+kubectl logs <driver pod name> secrets-store --since=1h | grep ^E
+```
+
+> It is always a good idea to include relevant logs from Azure Key Vault provider and Secrets Store CSI Driver when opening a new issue.
+
+## Common Issues
+
+Common issues or questions that users have run into when using Azure Key Vault provider for Secrets Store CSI Driver are detailed below.
+
+### driver name `secrets-store.csi.k8s.io` not found in the list of registered CSI drivers
+
+If you received the following error message in the pod events:
+```bash
+Warning FailedMount 42s (x12 over 8m56s) kubelet, akswin000000 MountVolume.SetUp failed for volume "secrets-store01-inline" : kubernetes.io/csi: mounter.SetUpAt failed to get CSI client: driver name secrets-store.csi.k8s.io not found in the list of registered CSI drivers
+```
+
+It means the Secrets Store CSI Driver pods aren't running on the node where application is running. 
+- If you've installed the AKV provider using deployment manifests, then make sure to follow the [instructions](../getting-started/installation) to install the Secrets Store CSI Driver. 
+- If you've already deployed the Secrets Store CSI Driver, then check if the node is tainted. If node is tainted, then redeploy the Secrets Store CSI Driver and Azure Key Vault provider by adding toleration for the taints.
+- If your application is running on windows node, then make sure to install the Secrets Store CSI Driver and Azure Key Vault provider on windows nodes by using the helm configuration values.
+
+### failed to get key vault token: nmi response failed with status code: 404

```

Comment:
Can you also add another section for https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/303:

```
failed to
find provider binary azure, err: stat /etc/kubernetes/secrets-store-csi-providers/azure/provider-azure: no such file or directory
```

Reply:
Done!